lightningd/invoice.c

```diff
@@ -720,6 +720,8 @@ static void gossipd_incoming_channels_reply(struct subd *gossipd,
 	json_add_u64(response, "expires_at", details->expiry_time);
 	json_add_string(response, "bolt11", details->bolt11);
 
+	notify_invoice_creation(info->cmd->ld, *info->b11->msat, info->payment_preimage, info->label);
```

Comment:
I kept it uniform without how all the other notifications are being called. I didn't want to introduce a bigger change. However I do agree with you, it would bet better to pass the entire `* info` to all notifications